src/app/rooms/rooms.component.html

```diff
@@ -0,0 +1,8 @@
+<h2>All Rooms</h2>
+<ul class="rooms">
+  <li *ngFor="let room of rooms">
+    <a routerLink="/detail/{{room.id}}">
```

Comment:
Does this router work?

Reply:
No. It does not work. But I need this for future testing on the room details component. I don't want to come back to room component to add this line of code for future testing.

Reply:
If this doesn't do anything, this shouldn't be here.

Reply:
Okay

Reply:
Yes, it does navigate you to that pages except for the fact that the page does not show anything and creates an error message in the console.